build/shade/_dotnet-test.shade

```diff
@@ -26,7 +26,7 @@ default configuration = 'Debug'
         noParallelTestProjects.UnionWith(NO_PARALLEL_TEST_PROJECTS.Split((char)','));
     }
 
-    var testArgs = " --configuration " + configuration;
+    var testArgs = " -f netcoreapp1.0 --configuration " + configuration;
```

Comment:
The new CLI expects the test runner (`dotnet-test-xunit`) to be a shared dependency because the test command is shared. Options here are 
(a) Change test project.jsons to commonize `dotnet-test-xunit` rather than list it as a `netcoreapp1.0` dependency
(b) Only run for netcoreapp1.0 for now and do `(a)` later (after we've unblocked the gate)